docroot/sites/all/themes/ilr_theme/scss/layout/_ilr-at-70-homepage-layout.scss

```diff
@@ -52,7 +51,7 @@ body.subsite-front {
           @include grid-span(7,6);
         }
         ul {
-          left: 42%;
```

Comment:
Can the social sharing code be changed so the markup is added after field-name-field-full-name instead of the h2? That would help the positioning of the social icons on the featured reflection. Thanks.